features/steps/tools.py

```diff
@@ -175,15 +174,39 @@ def check_volumes_created(client, test_name):
 
 
 def collect_ips(client):
-    print("collecting ips")
-    ports = client.network.ports()
     ips = []
-    for port in ports:
-        for fixed_ip in port.fixed_ips:
-            ips.append(fixed_ip["ip_address"])
+    print("collecting ips")
+    floating_ips = client.network.ips()
+    print("floating")
+    for ip in floating_ips:
+        ips.append(ip.floating_ip_address)
+        print(ip.floating_ip_address)
+    # ports = client.network.ports()    
+    # for port in ports:
+    #     for fixed_ip in port.fixed_ips:
+    #         ips.append(fixed_ip['ip_address'])
     return ips
 
 
+def collect_jhs(client, test_name):
+    servers=client.compute.servers()
+    lookup= test_name+"-jh"
+    lookup="default-jh" # just for testing delete
```

Comment:
do you know that it will take only the second variable declaration?

Reply:
yes it's for testing purposes, until we have an agreement on the naming conventions of the jh